assets/store/Conversation.js

```diff
@@ -53,6 +53,7 @@ export default class Conversation extends Reactive {
 
     if (params.conversation_id) {
       this.prop('persist', 'wantNotifications', this.is_private, {key: params.conversation_id +  ':wantNotifications'});
+      this.prop('rw', 'input', params.input || '');
```

Comment:
Maybe we can call it "keyboardInput" or "userInput"? Naming it "input" makes it bit hard to understand what it's really used for.

Also no need to do read default from `params`. Just default to `''`.

Reply:
Fixed (userInput)